docs/source/nlp/distillation.rst

```diff
@@ -0,0 +1,58 @@
+.. _megatron_distillation:
+
+Distillation
+==========================
+
+Knowledge Distillation (KD)
+--------------------------------
+
+KD involves using information from an existing trained model to train a second (usually smaller, faster) model, thereby "distilling" knowledge from one to the other.
+
+Distillation has two primary benefits: potentially faster convergence and/or higher end accuracy than traditional training.
+
+In NeMo, distillation is enabled by the `NVIDIA TensorRT Model Optimizer (ModelOpt) <https://github.com/NVIDIA/TensorRT-Model-Optimizer>`_ library -- a library to optimize deep-learning models for inference on GPUs.
+
+The logits-distillation process consists of the following steps:
+
+1. Loading both student and teacher model checkpoints using an appropriate parallelism strategy
```

Comment:
perhaps mention student and teacher need to have identical parallelism strategies?